src/DotNet.Status.Web/Controllers/AzurePipelinesController.cs

```diff
@@ -361,6 +400,125 @@ private async Task<string> BuildChangesMessage(Build build)
             return b.ToString();
         }
 
+        private async Task<string> BuildChangesMessage(string projectName, int buildId, bool mentionAuthors = false)
+        {
+            (BuildChange[] changes, int truncatedChangeCount) = await Client.GetBuildChangesAsync(projectName, buildId, CancellationToken.None);
+            return await BuildChangesMessage(changes, truncatedChangeCount, mentionAuthors);
+        }
+
+        private async Task<string> BuildChangesMessage(string projectName, int buildId, int compareBuildId, bool mentionAuthors = false)
```

Comment:
Do we really call both of these overloads?

Reply:
I think so? I think there was a reason I made both (tbh, I just looked at the fact that I had definitely written tests, and converted this to a real PR)